packages/backend/src/workers/action.ts

```diff
@@ -19,8 +19,14 @@ type JobData = {
 export const worker = new Worker(
   'action',
   tracer.wrap('workers.action', async (job) => {
-    const { stepId, flowId, executionId, nextStep, executionStep } =
-      await processAction({ ...(job.data as JobData), jobId: job.id })
+    const {
+      stepId,
+      flowId,
+      executionId,
+      nextStep,
+      executionStep,
+      computedParameters,
```

Comment:
Not sure I agree with this approach, since `computedParameters` will only ever be used for delay.

Since there's an edge case below for delay, was thinking we can re-use that. Off the top of my head, is it a viable  approach for `delayAsMiliseconds` to parse delay's `dataOut` or something?

Reply:
but isn't it weird or pointless to not use `computedParameters` if it's already being passed as an output field for `processAction`?

Reply:
As discussed (putting here in case we forget why we made some decisions) - this feels like a leaky abstraction and is not generic. Computed parameters only makes sense if processAction's caller knows exactly what step is being run, but from a software design point, I'm thinking it doesn't make sense for the caller to care about the running step.

Reply:
after researching a bit, i think a possibility is to use the step's lastExecutionStep data in, but seems like also not super generic?

Reply:
ya sounds good to me. Thinking we can just pass in `executionStep` inside the worker file like this, then you can whack dataIn, dataOut etc etc whatever you need:

```
    if (step.appKey === 'delay') {
      jobOptions = {
        ...DEFAULT_JOB_OPTIONS,
        delay: delayAsMilliseconds(step, executionStep),
      }
    }
```